pytest_jira.py

```diff
@@ -162,6 +162,8 @@ def __init__(
             self.basic_auth = None
 
     def _jira_request(self, url, method='get', **kwargs):
+        if 'verify' not in kwargs:
+            kwargs['verify'] = self.verify
```

Comment:
After closer investigation, it accepts true as well. in such case it uses system configuration.
So I think it is OK.